httputil/json.go

```diff
@@ -25,3 +25,12 @@ func WriteJSONResponse(w http.ResponseWriter, code int, resp interface{}) error
 	}
 	return nil
 }
+
+func WriteError(w http.ResponseWriter, code int, msg string) error {
```

Comment:
> I think it's nice to have a wrapper though.

Yeah, some sort of wrapper would be ok, but this isn't really a wrapper now. I'd like to see it more generic, or just leave it out.